src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/HTMLCSharpLanguageServerFeedbackFileLoggerProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Feedback
+{
+    [Shared]
+    [Export(typeof(HTMLCSharpLanguageServerFeedbackFileLoggerProvider))]
+    internal class HTMLCSharpLanguageServerFeedbackFileLoggerProvider : ILoggerProvider
+    {
+        private readonly FeedbackFileLoggerProvider _loggerProvider;
+
+        [ImportingConstructor]
+        public HTMLCSharpLanguageServerFeedbackFileLoggerProvider(
+            HTMLCSharpLanguageServerFeedbackFileLoggerProviderFactory loggerFactory)
+        {
+            if (loggerFactory == null)
+            {
+                throw new ArgumentNullException(nameof(loggerFactory));
+            }
+
+            _loggerProvider = (FeedbackFileLoggerProvider)loggerFactory.GetOrCreate();
```

Comment:
Hmm since these both use the same GetOrCreate() don't they both log to the same file path? I'd imagine you'd have access exceptions trying to write logs to the same file.

Reply:
Hmm good catch. They did write out to two separate files, however the file names are created by timestamp (and they were created ~5 seconds apart. 

https://github.com/dotnet/aspnetcore-tooling/blob/30f32f22fea230862ac888138e82c2ecb55c6936/src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/DefaultFeedbackFileLogWriter.cs#L101-L105

Updating so there's some additional identifier to ensure it's not possible for both logs to be written to the same file.

Reply:
Hmm, actually shouldn't be an issue due to our inbuilt retry logic. 

https://github.com/dotnet/aspnetcore-tooling/blob/30f32f22fea230862ac888138e82c2ecb55c6936/src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/DefaultFeedbackFileLogWriter.cs#L98-L99

Would you still prefer an identifier? Trade off is additional complexity vs easier accessibility. Thoughts?

Reply:
Lets definitely have an identifier only because it'll be a guessing game to know which log is what when investigating user issues

Reply:
Updated!

![image](https://user-images.githubusercontent.com/14852843/104526551-ad100980-55d0-11eb-8411-66396b2aa984.png)